pkg/agent/templates/templates.go

```diff
@@ -300,7 +300,8 @@ skip_verify = true
 {{ end }}
 [host]
 {{ range $e := .Endpoints -}}
-[host."{{ $e.URL }}"]
+[host."{{ printf "%s://%s:%s%s" $e.URL.Scheme $e.URL.Hostname() $e.URL.Port() $e.Path }}"]
```

Comment:
this is a fairly long-winded way of removing `[]` from IPv6 literals in the server URL.